### PR TITLE
doc: update apps.md

### PR DESCRIPTION
**Pull request by de-sh**
Closes #

### Why?
apps.md has been deemed outdated

### CODE REVIEW

File: docs/apps.md [hunk 1]
```diff
@@ -61,11 +74,20 @@ An example success response to an action with the id `"123"`, would look like:
...
+> actions = [{ name = "abc", timeout = 300 }] # Allow the connected app to send responses for action abc upto 5 minutes from receive, send a failure response and drop all responses afterwards if not yet completed.
+> ```
 
 ## Demonstration
 We have provided examples written in python and golang to demonstrate how you can receive Actions and reply back with either data or responses. You can checkout the examples provided in the `demo/` directory and execute them as such:
-1. Ensure uplink is running on device and connected to relevant broker.
+1. Ensure uplink is running on the device, connected to relevant broker and using the following config:
+```toml
+[tcpapps.main_app]
+actions = [{ name = "update_firmware" }, { name = "reboot" }, { name = "update_config" }]
```
Comment (de-sh) [inline docs/apps.md:89]:
```suggestion
port = 5555
actions = [{ name = "update_firmware" }, { name = "reboot" }, { name = "update_config" }]
```

**Review by de-sh — COMMENTED** [review]
Please include change

File: docs/apps.md [hunk 2]
```diff
@@ -61,11 +74,21 @@ An example success response to an action with the id `"123"`, would look like:
...
+> **NOTE:** There is a timeout mechanism which on being triggered will send a ***Failed*** response to platform and stop forwarding any *Progress* responses from the connected applications. In order to not trigger this timeout, an application must send a ***Failed*** or ***Completed*** response before the action timeout. Once an action has timedout, a failure response is sent and all it's future responses are dropped. Action timeouts can be configured per action when setting up uplink, as follows:
+> ```
+> [tcpapps.main_app]
+> port = 5555
+> actions = [{ name = "abc", timeout = 300 }] # Allow the connected app to send responses for action abc upto 5 minutes from receive, send a failure response and drop all responses afterwards if not yet completed.
```
Comment (amokfa) [inline docs/apps.md:81]: Can we confirm the docs match uplink behavior?

Comment (de-sh) [reply docs/apps.md:81]: Screenshot from 2023-06-13 20-05-21
Screenshot from 2023-06-13 20-05-00

**Pull request merged**